docs/en/src/README.md

```diff
@@ -0,0 +1,43 @@
+# Introduction
+
+## Supported Platforms
+
+| OS             | Core | GUI | ROS | ROS2 |
+| -------------- | ---- | --- | --- | ---- |
+| Linux (Ubuntu) | ✔    | ✔   | ✔   | ✔    |
+| MacOS          | ✔    | ✔   | ✔   |      |
+| Windows        | ✔    | ✔   |     |      |
+
+Ubuntu is supported as Linux. The approximate functionality is available even under MacOS and Windows.
+
+It can be used without ROS installation even when using Linux or MacOS.
+
+## Installation for Rust
+
+If you are new to Rust, please follow [this](https://www.rust-lang.org/tools/install) to install Rust.
+
+## Installation for latest OpenRR
+
+You can clone from [GitHub](https://github.com/openrr/openrr).
+
+```bash
+git clone https://github.com/openrr/openrr
+```
+
+## Installation for dependency
+
+Install software dependencies.
+
+```bash: Linux
```

Comment:
This annotation does not seem to be respected.

<img width="774" alt="linux" src="https://user-images.githubusercontent.com/43724913/204220314-c2afd22e-943c-4ad4-9fef-764978e425da.png">